src/js/stories/TableStories.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import { Table } from '../components/Table';
+
+const headers = ['A','B','C'];
+const headers2 = [
+    { className: 'red', label: 'A' },
+    { className: 'blue', label: 'B' },
+    { className: 'green', label: 'C' }
+];
+
+storiesOf('Table', module)
+    .add('Standard', scenario1)
+    .add('Hover Highlight', scenario1Hover)
+    .add('No Divider', scenario1NoDivider)
+    .add('No Default Padding', scenario1NoPadding)
+    .add('Table class', scenario2)
+    .add('Header class', scenario3)
+    .add('Fluid columns', scenario4)
+    .add('Fixed columns', scenario5);
+
+
+// SHORYUKEN
+const detailedHeaders = ['Name', 'DOB', 'Height', 'Weight', 'Blood Type'];
+const detailedData = [
+    [ 'Ryu',      '21-7-1964',  '5\'10"', '150 lbs', 'O' ],
+    [ 'E. Honda', '3-11-1960',  '6\'2"',  '304 lbs', 'A' ],
+    [ 'Blanka',   '12-2-1966',  '6\'5"',  '218 lbs', 'B' ],
+    [ 'Guile',    '23-12-1960', '6\'1"',  '191 lbs', 'O' ],
+    [ 'Ken',      '14-2-1966',  '5\'10"', '169 lbs', 'B' ],
+    [ 'Chun-Li',  '1-3-1968',   '5\'8"',  '-',       'A' ],
+    [ 'Zangief',  '1-6-1956',   '7\'',    '256 lbs', 'A' ],
+    [ 'Dhalsim',  '22-11-1962', '5\'10"', '107 lbs', 'O' ]
+];
+
+function renderRow(values) {
+    return (
+        <tr>
+            { values.map(value => <td>{value}</td>) }
```

Comment:
Perhaps add `key` attribute to the `td` (also possibly the `tr`) to avoid the https://fb.me/react-warning-keys warning in console.